examples/recsys2020.ipynb

```diff
@@ -36,17 +47,19 @@
     "import glob\n",
     "import gc\n",
     "\n",
-    "import cupy\n",
-    "import cupy as cp\n",
-    "import cudf\n",
-    "import rmm\n",
-    "import dask\n",
-    "import dask_cudf\n",
+    "import cupy                # CuPy is an implementation of NumPy-compatible multi-dimensional array on GPU\n",
```

Comment:
Do we need to import cupy twice (as cupy and cp)?  Can we remove this import and then change any instances of cupy to cp later?

We definitely don't need the same comment explaining what cupy is twice even if we leave the imports as is

Reply:
Thanks, I kept the import cupy as cp